packages/renderer/MdToHtml/rules/mermaid.ts

```
@@ -3,7 +3,7 @@ import { RuleOptions } from '../../MdToHtml';
 export default {
 
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-	assets: function(theme: any) {
+	assets: function (theme: any) {
```

ESLint

[error] Unexpected space before function parentheses.

(space-before-function-paren)